src/prefect/serialization/result_handlers.py

```diff
@@ -24,6 +24,20 @@ class Meta:
         object_class = ResultHandler
 
 
+class CustomResultHandlerSchema(ObjectSchema):
```

Comment:
Does this work as a catch all for any custom handlers someone may build?

Reply:
Yea, and honestly it doesn't do much --> all it produces is the name of the class they are using, nothing more.  This is _purely_ so that people can deploy Flows to Cloud with custom result handlers and
- it won't error out, because the handler can be "serialized"
- the name of the handler can be made visible in the DB / UI